sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/AddParticipantOptions.java

```diff
@@ -77,6 +91,17 @@ public AddParticipantOptions setOperationContext(String operationContext) {
         return this;
     }
 
+    /**
+     * Set the call back URI override.
+     *
+     * @param callbackUriOverride The call back URI override to set
+     * @return the AddParticipantOptions object itself.
+     */
+    public AddParticipantOptions setCallbackUriOverride(String callbackUriOverride) {
```

Comment:
There are still some section where it uses ...Uri. We should all have Url throughout for uniformity